tools/ingest-json-to-grok/src/main/java/org/logstash/ingest/JsonToGrok.java

```diff
@@ -0,0 +1,41 @@
+package org.logstash.ingest;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import javax.script.Invocable;
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+
+/**
+ * Ingest JSON DSL to Logstash Grok Transpiler.
+ */
+public final class JsonToGrok {
```

Comment:
Maybe just shorten this to Grok. We know we're in the ingest package at this point.